src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -134,6 +139,9 @@ protected override ILspServices ConstructLspServices()
 
         static void AddHandlers(IServiceCollection services)
         {
+            services.AddRegisteringHandler<ImplementationEndpoint>();
+            services.AddRegisteringHandler<SignatureHelpEndpoint>();
```

Comment:
I guess this means I don't get a donut... but fantastic that we have a test to find them all now <3